decryptor/postgresql/data_encoder_test.go

```diff
@@ -151,15 +162,15 @@ func TestTextMode(t *testing.T) {
 		// decoder expects valid string and pass as is, so no errors. but on encode operation it expects valid int literal
 		{input: []byte("some data"), decodedData: []byte("some data"), encodedData: []byte("some data"),
 			decodeErr: nil, encodeErr: nil,
-			setting:    &config.BasicColumnEncryptionSetting{TokenType: "int32", DataType: "int32"},
+			setting:    &config.BasicColumnEncryptionSetting{TokenType: "int32", DataType: "int32", DataTypeID: pgtype.Int4OID},
```

Comment:
hm... I think we should not allow to specify both values: data_type and data_type_oid. IMHO users should specify only one of them. To avoid misconfiguration and so on. and as I see, we have special error `ErrDataTypeWithDataTypeID` for that. But here, as I see, we have decodeErro and encodeErr.